examples/lib/dynamic_outputs.exs

```diff
@@ -1,7 +1,8 @@
-alias Membrane.VideoCompositor.Examples.DynamicOutputs.Pipeline
+alias Membrane.VideoCompositor.Examples.DynamicOutputs.Pipeline, as: DynamicOutputsPipeline
```

Comment:
Didn't notice that before, but these names can awaken Java PTSD. Let's rename `Membrane.VideoCompositor.Examples.DynamicOutputs.Pipeline` to `DynamicOutputs.Pipeline` or `DynamicOutputsPipeline` and don't alias at all

Reply:
And you can move the contents of `dynamic_outputs/pipeline.ex` into this script, I don't see a benefit for having this split into multiple files